sdk/src/driftClient.ts

```diff
@@ -5348,6 +5417,29 @@ export class DriftClient {
 		}[],
 		marketIndexes: number[]
```

Comment:
i think there are some bots where we wouldn't want this check, maybe we can add a flag here to filter out invalid oracles. or just do it outside fn